apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget_Configuration.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Logging.V2;
+using Google.Protobuf.WellKnownTypes;
+using NLog.Config;
+using NLog.Layouts;
+using NLog.Targets;
+using System;
+using System.Collections.Generic;
+
+namespace Google.Cloud.Logging.NLog
+{
```

Comment:
Maybe add a `//` (not `///`) comment for why this is separated?

Reply:
Done